packages/web-wallet/src/components/__tests__/SendDialog.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { describe, it, expect } from 'vitest';
```

Comment:
This file does not test anything from the `SendDialog` but at least it is testing something from outside the test, but why test something from outside the repository?